app/v1/views/auth.py

```diff
@@ -28,8 +27,6 @@ def user_signup():
         response = jsonify({"Error": "Fields should not be empty"})
```

Comment:
Each field should be validated with the most appropriate error message(s) being returned for that(those) field(s). The validation functions should be abstracted from the views.